plugins/sync.js

```diff
@@ -83,73 +84,93 @@ window.plugin.sync.RegisteredMap = function(options) {
   this.callback = options['callback'];
   this.initializedCallback = options['initializedCallback'];
   this.authorizer = options['authorizer'];
-  this.uuid = options['uuid'];
+
+  this.uuid = options['uuid'];                              // local UUID
+  this.lastUpdateUUID = null;                               // the UUID that we think
+                                                            // did last Update (can be us)
+  this.remoteUUID = null;                                   // the remote UUID in drive
 
   this.intervalID = null;
   this.map = null;
-  this.lastUpdateUUID = null;
   this.dataStorage = null;
 
   this.forceFileSearch = false;
   this.initializing = false;
   this.initialized = false;
   this.failed = false;
-
+  this.doSync = 'remote';                                   // normal 'sync'
+                                                            // 'no' sync needed
+                                                            // force 'remote' (1st sync)
+                                                            // force 'local'
+                                                            // sync is on 'stop'
+  
```

Comment:
I don't understand the comment. I assume "sync","no","remote","local","stop"  are valid states of `doSync` ?

maybe better something like - just my guess what they could mean:
```suggestion
 /** 
 *  doSync : state of current sync
 *   = "sync" : normal sync (what ever this means)
 *   = "no" : sync needed (or "no sync running, start delay" ?)
 *   = "stop" : sync stopped by user
 *   = "remote" : get values from remote
 *   = "local : use local values instead of remote
 */
  this.doSync = 'remote';
```